apps/brainstorm/build.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-VERSION=0.0.2
+VERSION=0.0.3
 APP_NAME="brainstorm"
 APP_VERSION="latest"
```

Comment:
Should the `APP_VERSION` be `R2023a` then?

Reply:
Nop, R2023a is the maltab runtime, brainstorm is evolving and sometimes they change the matlab runtime version that we need.

Reply:
Unfortunately, it's not possible to download a specific version of brainstorm. It's only possible to download the latest version, and then tag it (after launching it locally to check its "date" and use that as the tag). For the HIP, we haven't done that, we used `latest`. I'm against using `latest`, this does not work well with k8s.

Reply:
okay, I see. Thanks.

Reply:
So in the case of apps where we used to put latest we use only the chart version number then ?

Reply:
Actually using the date of the brainstorm build isn't a great idea, because if we rebuild the image, it will change. Maybe we should use `today` instead, what do you think?

Reply:
That works for me , @greut ?

Reply:
the rebuilding the image issue can be blocked at the registry side, https://goharbor.io/docs/1.10/working-with-projects/working-with-images/create-tag-immutability-rules/
we can stick to 0.0.x for now, I'd say.

Reply:
@iDmple , works for you ?

Reply:
sure